app/views/practices/planning_checklist.html.erb

```diff
@@ -1,164 +0,0 @@
-<% provide :head_tags do %>
-  <%= javascript_include_tag '_practice_utilities', 'data-turbolinks-track': 'reload' %>
-<% end %>
-
-<%= render 'breadcrumbs' %>
-
-<section class="margin-top-7 margin-bottom-6" id="next-steps-top">
-  <div class="grid-container">
-    <%= render partial: "shared/messages", locals: {small_text: false} %>
-    <div class="grid-row flex-align-end margin-bottom-105">
-      <h1 class="font-sans-2xl margin-right-3 margin-y-0 grid-col-auto grid-col-auto">Adoption checklist</h1>
-      <div class="grid-col-fill share-checklist-container margin-bottom-05">
-        <a target="_blank"
-           href="mailto:?subject=<%= email_checklist_subject(@practice) %>&body=<%= email_checklist_body(@practice) %>" class="link-primary text-super">
-          <i class="fas fa-external-link-alt"></i>&nbsp;
-          Share checklist
-        </a>
-      </div>
-    </div>
-    <p class="margin-0 font-sans-lg grid-col-10 line-height-36">
-      Here is a step-by-step guide to give you the best chance of success for adopting this practice. After completing these
-      preparatory steps, we will put you in contact with the practice team members for full adoption details.
-    </p>
-  </div>
-</section>
-
-
-<!-- Checklist -->
-
-<section>
-  <%= form_for :commit, url: {action: :commit }, method: :post do %>
```

Comment:
I believe since we are getting rid of this file and this `:commit` action we can also remove all the functionality related to`:commit` and `:committed` in the practices_controller as well as the routes and mailer code... if that makes sense.

Reply:
Yes - i removed the can_view_commited methods and before action for the :commit as well as updated methods in the application_helper and coverage index files.